src/when-you-get-an-alert.html

```diff
@@ -72,7 +72,10 @@ <h2 class="govuk-heading-m">
         If you want to see an alert again
       </h2>
       <p class="govuk-body">
-        You can find current and past alerts on your phone or tablet.
+        Visit gov.uk/alerts to see <a href="/alerts/current-alerts">current alerts</a> and <a href="/alerts/current-alerts">past alerts</a>.
+      </p>
+      <p class="govuk-body">
+        You can also find them on your phone or tablet. 
```

Comment:
If we change line 75, we should update this to:

```suggestion
        You can also search for them on your phone or tablet. 
```